README.md

```diff
@@ -41,6 +41,25 @@ press enter.
 On OpenBSD the `i3lock` binary needs to be setgid `auth` to call the
 authentication helpers, e.g. `/usr/libexec/auth/login_passwd`.
 
+Building i3lock
+---------------
+To build i3lock, simply run these commands:
+```
+apt build-deps i3lock
```

Comment:
Oh yeah my bad for the warning, forgot that!

I thought it would be nice to have example instructions to install dependencies, but I understand it's hard to cover all use cases and hard to maintain.